### ISSUE TITLE
picker dos not show negative number

### ISSUE THREAD

**Issue by kia1349** [ShawnLin013/NumberPicker#23]
hi ShawnLin013

i have a fragment with numberpicker
i call fragment and pass   alarm.npminutes to it;
it has negative integer value , for example -19 
this is my code:

```  
  private void showNpminutesDialog(final Alarm alarm) {
        final FragmentTransaction ft = getFragmentManager().beginTransaction();
        final Fragment prevx = getFragmentManager().findFragmentByTag("npminutes_dialog");
        if (prevx != null) {
            ft.remove(prevx);
        }
        ft.addToBackStack(null);
        /* Create and show the dialog.*/
        final npminutesDialogFragment newFragment =
                npminutesDialogFragment.newInstance(alarm, alarm.npminutes, getTag());
        newFragment.show(ft, "npminutes_dialog");
    }
```

in fragment npminutesDialogFragment , i read alarm.npminutes by below ode:

        final int npminutes = bundle.getInt(KEY_NPMINUTES);

then i create dialog with numberpicker as below

```
       View v1x = getActivity().getLayoutInflater().inflate(R.layout.numberpicker, null);
        picker = (NumberPicker) v1x.findViewById(R.id.horizontal_number_picker);
        picker.setOnValueChangedListener(new NumberPicker.OnValueChangeListener() {
            @Override
            public void onValueChange(NumberPicker picker, int oldVal, int newVal) {
                // do your other stuff depends on the new value
            }
        });
        picker.setValue(alarm.npminutes); // <==========   -19 
        picker.setMinValue(-59);
        picker.setMaxValue(59);
        picker.setWrapSelectorWheel(false);
        final AlertDialog alertDialog = new AlertDialog.Builder(context)
                .setView (v1x)
                .setPositiveButton(R.string.time_picker_set, new DialogInterface.OnClickListener() {
                    @Override
                    public void onClick(DialogInterface dialog, int which) {
                        set(alarm, tag);
                    }
                })
                .setNegativeButton(R.string.time_picker_cancel,
                        new DialogInterface.OnClickListener() {
                        @Override
                        public void onClick(DialogInterface dialog, int which) {
                            dismiss();
                        }
                })
                .setTitle("select plus/minus minute for alarm time")
                .create();
        return alertDialog;
    }
```

now when dialog opened with negative number , wheel show  20 instead -20 

![photo_2016-12-03_15-58-55](https://cloud.githubusercontent.com/assets/10990724/20859364/3ed2a3dc-b972-11e6-8798-115c164d8326.jpg)

![photo_2016-12-03_15-58-59](https://cloud.githubusercontent.com/assets/10990724/20859365/497a33ea-b972-11e6-968e-606fd8622b1d.jpg)

thanks alot for your library

**Comment by ShawnLin013** [comment]
Hi @kia1349,

It has been fixed at v2.3.0.
Thank you!

**Comment by kia1349** [comment]
thanks alot too shawnlin
